client/cluster_profile.go

```diff
@@ -207,6 +207,20 @@ func (h *V1Client) GetClusterProfile(uid string) (*models.V1ClusterProfile, erro
 	return resp.Payload, nil
 }
 
+// DownloadClusterProfileUIDSpc retrieves an existing cluster profile by UID.
+func (h *V1Client) DownloadClusterProfileUIDSpc(w *io.PipeWriter, profileUID string) (string, error) {
+	params := clientv1.NewV1ClusterProfilesUIDSpcDownloadParamsWithContext(h.ctx).
+		WithUID(profileUID)
+
+	resp, err := h.Client.V1ClusterProfilesUIDSpcDownload(params, w)
```

Comment:
This method returns an `io.ReadCloser` we need to actually read from it into a buffer and save the resulting data somewhere.

Reply:
my plan is to consume it like this 

<img width="689" alt="image" src="https://github.com/user-attachments/assets/25dd6911-0297-4e15-9f6e-9fe199bf190b" />


kind of like this in the stylus repo
https://github.com/spectrocloud/stylus/blob/7b29a7e908627516135c003e17ac988e7cd45750/pkg/services/spectro-cli/cmd/spectro-cli-build.go#L319

Reply:
Ok. `ExportClusterProfile` returns a `bytes.Buffer` and `DownloadLogs` returns an `io.Writer`. It would be good to standardize these.